rpm/ngfd-plugin-native-vibrator.spec

```diff
@@ -0,0 +1,46 @@
+Name:       ngfd-plugin-native-vibrator
+Summary:    Droid Vibrator native plugin for ngfd
+Version:    0.92
+Release:    1
+Group:      System/Daemons
+License:    LGPL 2.1
```

Comment:
The right formatting for this is LGPLv2.1 or LGPLv2+ depending if there is "or later mentioned in the license text". Same issue with the old .spec I guess.